### PR TITLE
DISM - disable windows features

### PR DESCRIPTION
**Pull request by chrisbednarski**
**Background**

When using the Windows UI to turn on the FTP service, only the four features that are required are enabled
image

`dism.exe /online /get-features /format:table` shows these enabled
```
------------------------------------------- | -------
Feature Name                                | State
------------------------------------------- | -------
IIS-WebServerRole                           | Enabled
IIS-FTPServer                               | Enabled
IIS-FTPSvc                                  | Enabled
IIS-FTPExtensibility                        | Enabled
```

When trying to enable the WebServerRole feature using DISM (10.0.19041.844), I get multiple other IIS features turning on, even though DISM is asked to enable the features without their parent dependencies.

`dism.exe /online /enable-feature /featurename:IIS-WebServerRole`  (without using the `/All` flag) shows these enabled (on Windows 10.0.19045.2965 image)
```
IIS-WebServerRole                           | Enabled
IIS-WebServer                               | Enabled
IIS-CommonHttpFeatures                      | Enabled
IIS-HttpErrors                              | Enabled
IIS-Security                                | Enabled
IIS-RequestFiltering                        | Enabled
IIS-HealthAndDiagnostics                    | Enabled
IIS-HttpLogging                             | Enabled
IIS-Performance                             | Enabled
IIS-WebServerManagementTools                | Enabled
IIS-StaticContent                           | Enabled
IIS-DefaultDocument                         | Enabled
IIS-DirectoryBrowsing                       | Enabled
IIS-HttpCompressionStatic                   | Enabled
IIS-ManagementConsole                       | Enabled
```

**Description**
Added `RemoveFeature` property to the CA that holds a regex expression for features to be removed after all features resolved from `EnableFeature` regex have been enabled

`<PanelSW:Dism EnableFeature="^IIS-WebServerRole$" EnableAll="no" RemoveFeature="^IIS-(?!(?:WebServerRole)$)\w+$"/>`

will result in 
```
------------------------------------------- | ---------------
Feature Name                                | State
------------------------------------------- | ---------------
IIS-WebServerRole                           | Enabled
IIS-HttpErrors                              | Disable Pending
```

`RemoveFeature` is gentle on windows features that were enabled prior to running the MSI installer. Those features will remain enabled, unless `ForceRemove` flag is set to `yes/true`

### CODE REVIEW

File: wixlib/en-US.wxl [hunk 1]
```diff
@@ -32,6 +32,7 @@
   <String Overridable="yes" Id="DismTemplate" Value="Enabling [1]" />
   <String Overridable="yes" Id="DismPackageFailure" Value="Failed adding a Windows Feature package [2]:&#xA;[3]" />
   <String Overridable="yes" Id="DismFeatureFailure" Value="Failed enabling Windows Feature [2]:&#xA;[3]" />
+  <String Overridable="yes" Id="DismUnwantedFeatureFailure" Value="Failed disabling Windows Feature [2]:&#xA;[3]" />
```
Comment (chrisbednarski) [inline wixlib/en-US.wxl:36]: missing `DismUnwantedFeatureFailure` from `wixlib/he-IL.wxl`

Comment (nirbar) [reply wixlib/en-US.wxl:36]: Hebrew text:
```
  <String Overridable="yes" Id="DismFeatureFailure" Value="כשלון בהסרת מאפייני מערכת [2]:&#xA;[3]" />
```

**Comment by nirbar** [comment]
@chrisbednarski Did you notice my comments on the PR?

**Comment by chrisbednarski** [comment]
> @chrisbednarski Did you notice my comments on the PR?

I thought I saw and addressed all your comments??
Now I'm not sure

**Comment by nirbar** [comment]
@chrisbednarski Scroll down in the 'Files changed' tab, you'll see the comments there
image

**Comment by chrisbednarski** [comment]
The only one I see there is this.

image

And I added the translation in

**Comment by nirbar** [comment]
image
image
image
image

**Comment by nirbar** [comment]
@chrisbednarski Thanks again for your contribution

**Comment by nirbar** [comment]
@chrisbednarski Nuget package [PanelSwWixExtension4.3.7.0.221](https://www.nuget.org/packages/PanelSwWixExtension4/3.7.0.221) contains your changes

**Comment by chrisbednarski** [comment]
Thanks

**Pull request merged**